applications/GeoMechanicsApplication/tests/cpp_tests/test_newmark_Pw_scheme.cpp

```diff
@@ -0,0 +1,141 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_strategies/schemes/newmark_quasistatic_Pw_scheme.hpp"
+#include "spaces/ublas_space.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
+using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+
+namespace Kratos::Testing {
+
+namespace {
+NewmarkQuasistaticPwScheme<SparseSpaceType, LocalSpaceType> CreateValidScheme()
+{
+    NewmarkQuasistaticPwScheme<SparseSpaceType, LocalSpaceType> result(0.75);
+    return result;
+}
```

Comment:
This function has been inlined in one of the other test files. You may consider to do the same thing here.

Reply:
Fine with me, done